code/game/objects/items/storage/toolbox.dm

```diff
@@ -86,6 +86,24 @@
 
 /obj/item/storage/toolbox/mechanical/old/heirloom/PopulateContents()
 	return
+	
+/obj/item/storage/toolbox/mechanical/old/clean
+	name = "mechanical toolbox"
+	desc = "A blue, old toolbox, it looks robust."

```

Comment:
The name and description should be identical to a regular old heirloom toolbox.

Reply:
Should it really say "It's seen better days" when the toolbox is in pristine condition.
As for the name I'll change that to just 'toolbox'.

Reply:
The sprite has seen better days when it was a common item

Reply:
wouldn't it be better to have a description to indicate it's robust?

Reply:
idc lol it's bikeshed

Reply:
ill just close this review then